### DISCUSSION TITLE
Content not rendering

### DISCUSSION THREAD

**Discussion by mojowill** [filamentphp/filament#14143] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I've installed Filament into an existing project where we were already using tables, forms etc but not the panels.

After simply installing panels following the guide I can access the route as expected but some parts of the page simply don't seem to render. I can see the correct coloured background and can even see some slightly skewed items items in the nav side bar and when inspecting I can see the HTML is present but it's not being rendered.

I've not changed any config simply installed and tried to visit the default dashboard.

Also If I visit the default login page the submit button simply spins and when filling in the form and hitting enter I get a 405 method not allowed error. I removed `->login()` from my Panel Provider which then forwards me to my own application login page which works as expected and then redirects me to my filament dashboard after authenticating however still the page loads incorrectly as outlined above.

**Comment by leandrocfe** [comment]
Something is wrong in your project, but it is difficult to assist without any code or a clear context..

**Reply by mojowill** [reply]
Yeah I kind of knew it was project specific as a clean Breeze and Filament project is fine. My understanding though is that Filament is somewhat separate to all my application logic and even front end though so was unsure where the conflict might be?

**Reply by leandrocfe** [reply]
I recommend double-checking your Filament setup, ensuring that you’re using the correct Livewire version. 
You can also try to republish assets

`php artisan filament:upgrade`

**Reply by mojowill** [reply]
Assets are published correctly and are loading correctly as far as I can tell, no HTTP errors or anything. Something isn't happy but there is no indication as to what it might be.
